cmake/multi_image.cmake

```diff
@@ -27,6 +27,7 @@ if(IMAGE_NAME)
   share("set(${IMAGE_NAME}KERNEL_ELF_NAME ${KERNEL_ELF_NAME})")
   share("list(APPEND ${IMAGE_NAME}BUILD_BYPRODUCTS ${PROJECT_BINARY_DIR}/${KERNEL_HEX_NAME})")
   share("list(APPEND ${IMAGE_NAME}BUILD_BYPRODUCTS ${PROJECT_BINARY_DIR}/${KERNEL_ELF_NAME})")
+  share("set(${IMAGE_NAME}ZEPHYR_RUNNERS_YAML ${PROJECT_BINARY_DIR}/runners.yaml)")
```

Comment:
Does this mean we don't need the modification of `yaml_contents` in partition_manager.cmake? https://github.com/nrfconnect/sdk-nrf/blob/master/cmake/partition_manager.cmake#L317

Reply:
I believe that modification is due to the merging of mcuboot, spm, and app images.
Those are still flashed as one hex, so currently I believe the modification is still needed.

But if it is decided to flash all those images independently, then I believe this code could be removed, but I think care must be taken in that case regarding the signing and flashing of right signed image.

You and @hakonfam are probably the best people to consider this.